LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/BasicsStation/LnsOperation.cs

```diff
@@ -0,0 +1,123 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace LoRaWan.NetworkServer.BasicsStation
+{
+    using System.Diagnostics.Metrics;
+    using System.Net;
+    using System.Text.Json;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using LoRaTools;
+    using Microsoft.Extensions.Logging;
+
+    internal interface ILnsOperation
```

Comment:
nit: this represents any operations that can be triggered server side. Maybe we can use a name that represents that better: ILnsRemoteOperation, ILnsRemoteRequest, ILnsRemoteCall?

Reply:
Done in a104ae286a968e8188763bd849ddc6315a8b1202